src/sdkV2.ts

```diff
@@ -93,6 +93,29 @@ export default class CegaEvmSDKV2 {
     });
   }
 
+  async bulkSettleVaultsDcs(
+    vaultAddresses: EvmAddress[],
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.bulkSettleDCSVaults(vaultAddresses);
+  }
+
+  async bulkStartTradesDcs(
+    vaultAddresses: EvmAddress[],
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.bulkStartDCSTrades(vaultAddresses);
+  }
+
+  async submitDispute(vaultAddress: EvmAddress): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.submitDispute(vaultAddress);
+  }
+
+  /**
```

Comment:
are the above methods trading methods too?

Reply:
they're technically part of overall "trading flow" but are MM operations, i added their own comment section